v22.2/row-level-ttl.md

```diff
@@ -38,61 +36,110 @@ Once rows are expired (that is, are older than the specified [TTL interval](#par
 
 ## Syntax overview
 
-TTLs are defined on a per-table basis using SQL statements. The syntax for creating a table with an automatically managed TTL extends the [`storage_parameter` syntax](sql-grammar.html#opt_with_storage_parameter_list). For example, the SQL statement
+TTLs are defined on a per-table or per-row basis using SQL statements.
+
+- [Per-table TTL](#per-table-ttl)
+- [Per-row TTL](#per-row-ttl)
+
+### Per-table TTL
```

Comment:
The difference between `ttl_expire_after` and `ttl_expiration_expression` isn't per-table vs per-row. Both of them set TTL per-row in the table.

`ttl_expire_after` is a convenient way of setting rows to expire a fixed amount of time after they are created or updated.

`ttl_expiration_expression` lets you customize the expiration logic further by providing an expression. (You could get the same behavior as `ttl_expire_after` by creating a column with a default value and having the `ttl_expiration_expression` reference that column.)

Additionally, the 2 storage params can co-exist where the `ttl_expire_after` creates the `crdb_internal_expiration` column and `ttl_expiration_expression` overrides the default value of `crdb_internal_expiration`.

Reply:
Thanks for clarifying this! I have updated these sections of the doc to add this info - PTAL